data.sql

```diff
@@ -43,15 +43,15 @@ INSERT INTO vets (name, age, date_of_graduation) VALUES
   ('Jack Harkness', 38, '2008-06-08') 
 ;
 
-
+-- Insert Values into Specializations
 INSERT INTO specializations (vet_id, species_id) VALUES 
   (1, 1),
```

Comment:
- [ ] Please add the following queries that you used here in the `data.sql` file.
```sql
-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;

-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```